flash_all.sh

```diff
@@ -3,14 +3,20 @@ set -e
 
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
 cd $DIR
+EDL=$DIR/edl/edl
+
+flash() {
```

Comment:
instead of copy/pasting this function in all the scripts, let's put it in a script called `tools/edl_flash.sh`

Reply:
agree

Reply:
this is more what i was thinking: https://github.com/commaai/agnos-builder/pull/201/commits/82abb831316f82b50ae61594ae0d68d69c30ed90